src/components/StakeForm.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+const StakeForm = ({handleChange, handleSubmit, amount}) => (
+    <div className="container col-sm-4 col-offset-3">
+    <div className="card">
```

Comment:
Please move all the card related divs into a separate Card control, so that you are not repeating the pattern in multiple places